BridgeSDK/BridgeAPI/SBBUploadManager.m

```diff
@@ -427,43 +422,51 @@ - (void)uploadedFileToS3WithTask:(NSURLSessionUploadTask *)uploadTask response:(
 - (void)downloadedBridgeUploadSessionWithDownloadTask:(NSURLSessionDownloadTask *)downloadTask fileURL:(NSURL *)file
 {
     NSError *error;
+    NSString *uploadFileURL = downloadTask.taskDescription;
```

Comment:
not sure I get what you're concerned about here--if taskDescription did somehow get nil'ed, ARC would ensure it doesn't get released immediately because of this assignment, no?